gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -1022,13 +1024,6 @@ def main(args):  # noqa: D103
 
         if args.export_vcf:
             contig = f"chr{contig}" if contig else None
-
-            if contig and test:
```

Comment:
why drop this if statement?

Reply:
Added this back and the contig is now below the error